Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -141,7 +142,6 @@ export default class ReportsView extends Vue {
     private mounted() {
         this.logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
         this.loadName();
-        this.fetchMedicationStatements();
```

Comment:
func. test should not change code (just in case we merge these by mistake).

Reply:
this was a bug identified by the functional test, do you want me to create another PR just for the fix?

Reply:
@tiagograf-AOT  sounds good, I have raised it just in case we had missed something due to branches merging.